omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+********************
+Version requirements
+********************
+
+Support levels
+--------------
+
+The following sections use the terminology in the table below to
+describe the support status of a given component, as it progresses
+from being new and not supported, to supported and regularly tested,
```

Comment:
not sure if "frequently" captures your meaning better than "regularly" (or if either is true!)